src/boot/bootctl-install.c

```diff
@@ -830,7 +1087,19 @@ int verb_install(int argc, char *argv[], void *userdata) {
                                 return r;
                 }
 
-                r = install_binaries(arg_esp_path, arch, install);
+                r = install_binaries_extra(
+                                arg_esp_path,
+                                arch,
+                                install,
+                                &default_loader_path,
+                                &default_loader_description);
+                if (r == 0)
+                        r = install_binaries(
+                                        arg_esp_path,
+                                        arch,
+                                        install,
+                                        &default_loader_path,
+                                        &default_loader_description);
```

Comment:
this makes no sense to me. it appears we install "extra" XOR our own stuff. it should be AND, not XOR!

we always want our resources to be dropped in. The "extra" stuff is optional, and only in addition.